_posts/2021-05-04-ccc-webinar-recording.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: post
+date: 4 May 2021
+title: CCC webinar recording now available
+---
+
+On 29 April members of the Veracruz team participated in a CCC webinar, introducing the project.
```

Comment:
Maybe expand the acronym and/or add the [link](https://confidentialcomputing.io/)?

Reply:
Forgot about this PR.  This is done now.